From 5d63a9cf127a691daded1017509953c1cee5ad36 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Wed, 3 May 2017 11:06:30 +0800
Subject: [PATCH 1729/5242] MLK-14781-2: ARM: dts: change audio pll frequency
 for ssi master mode

commit  b5faa03e24535f4b661ab24acb30e5b98e9c458f from
https://source.codeaurora.org/external/imx/linux-imx.git

Original frequency 786.432MHz can't be divide to frequency 2.304MHz
which is needed by 48kHz/24bit/2channel, So update the pll frequency
for 24bit case.

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
Reviewed-by: Daniel Baluta <daniel.baluta@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qdl-sabresd.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
index ad1227e..4d92289 100644
--- a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
@@ -1092,7 +1092,7 @@
 	assigned-clock-parents = <&clks IMX6QDL_CLK_OSC>,
 				 <&clks IMX6QDL_CLK_PLL4>,
 				 <&clks IMX6QDL_CLK_PLL4_AUDIO_DIV>;
-	assigned-clock-rates = <786432000>, <0>, <0>;
+	assigned-clock-rates = <1179648000>, <0>, <0>;
 	status = "okay";
 };
 
-- 
1.7.9.5

